src/data/constants/courseOptions.js

```diff
@@ -0,0 +1,3 @@
+export const FAIL_COURSE_OPTIONS = 'FAIL_COURSE_OPTIONS';
+export const RECEIVE_COURSE_OPTIONS = 'RECEIVE_COURSE_OPTIONS';
+export const REQUEST_COURSE_OPTIONS = 'REQUEST_COURSE_OPTIONS';
```

Comment:
Why not continue the convention you updated the previous file to follow (i.e. define the constants and then export them)?